transformations/swap_characters/transformation.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class SwapCharactersPerturbation(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION,
+        TaskType.TEXT_TAGGING,
+    ]
+    languages = ["en"]
+
+    def __init__(self, seed=0):
+        super().__init__(seed)
+
+    def generate(self, sentence: str, prob=0.05):
+        pertubed = self.swap_characters(text=sentence, prob=prob, seed=self.seed)
+        return pertubed
```

Comment:
Please pull "main" once. There will be a minor change in the test cases format which the generate function returns multiple sequences now.

Reply:
As in it should return [perturbed] in a list? After looking at other transformations, that's my understanding. Pushed.